float32 x
float32 y
float32 z
string limb
---
bool poop